vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -48,7 +48,7 @@ type internal FSharpCheckerProvider
     ) =
     let checker = 
         lazy
-            let checker = FSharpChecker.Create()
+            let checker = FSharpChecker.Create(projectCacheSize = 200, keepAllBackgroundResolutions = false)
```

Comment:
@dsyme in https://github.com/Microsoft/visualfsharp/blob/ee84eb77bcd62379d8178e3167a17d12d204de30/src/fsharp/vs/service.fs I see things like this:

```fsharp
GetEnvInteger "FCS_ProjectCacheSizeDefault" 3
```

Does that mean that we can override the value with environment variables?

I think at least it would be a way to tweak those settings if situation becomes dire, although with this PR this is not possible for that 200 setting.